src/generators/static/README.md.hbs

```diff
@@ -64,13 +66,15 @@ For more information about how to create an Azure AD Application check out [this
 ```javascript
 const { {{ clientClassName }} } = require("{{ clientPackageName }}");
 const { DefaultAzureCredential } = require("@azure/identity");
+// For client-side applications running in the browser, use InteractiveBrowserCredential instead of DefaultAzureCredential. See https://aka.ms/azsdk/js/identity/examples for more details.
```

Comment:
maybe we can add a commented out example of how to use the interactiveBrowserCredential?

```typescript
 // const credential = new InteractiveBrowserCredential({
 //   tenantId: "<YOUR_TENANT_ID>",
 //   clientId: "<YOUR_CLIENT_ID>"
 // });
// const client = new {{ clientClassName }}(credential, subscriptionId);
```

Reply:
I like your idea, let me add the example :)